_includes/layouts/tools.html

```diff
@@ -0,0 +1,97 @@
+---
+layout: layouts/default
+---
+
+{% comment %}
+  This layout is used for individual posts / updates.
+
+  Default front matter is set in posts/posts.json. Individual
+  front matter and overrides of default values are available
+  in the front matter of specific posts in the posts directory.
+{% endcomment %}
+<!-- FOR MODAL FUNCTIONALITY - JavaScript Bundle with Popper -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/js/bootstrap.min.js" integrity="sha384-+sLIOodYLS7CIrQpBjl+C7nPvqq+FbNUBDunl/OZv93DB7Ln/533i8e/mZXLi/P+" crossorigin="anonymous"></script>
+<!-- FOR MODAL FUNCTIONALITY - JavaScript Bundle with Popper -->
+
+<!-- For Accessibility Checks -->
+<style>
+html > body > div > a, #menu-navigation > div > ul > li > a > span { color: #0072ed }
```

Comment:
are you adding these inline here because importing new stylesheets isn't working? if so, i'm working on that... 😃